src/instrumentsscene/view/instrumentsonscorelistmodel.cpp

```diff
@@ -153,17 +161,38 @@ void InstrumentsOnScoreListModel::loadOrders()
     TRACEFUNC;
 
     m_scoreOrders = repository()->orders();
-    emit ordersChanged();
 
     INotationPtr notation = context()->currentNotation();
-    QString currentOrderId = notation ? notation->parts()->scoreOrder().id : QString();
+    ScoreOrder currentOrder = notation ? notation->parts()->scoreOrder() : m_scoreOrders[0];
+    bool orderCustomized = currentOrder.customized;
+
+    if (!m_scoreOrders.contains(currentOrder)) {
+        currentOrder.customized = false;
+        m_scoreOrders.append(currentOrder);
+    }
+
+    emit ordersChanged();
 
     for (int i = 0; i < m_scoreOrders.size(); ++i) {
-        if (m_scoreOrders[i].id == currentOrderId) {
+        if (m_scoreOrders[i].id == currentOrder.id) {
             doSetCurrentOrderIndex(i);
             break;
         }
     }
+
+    if (orderCustomized) {
+        doSetCurrentOrderIndex(createCustomizedScoreOrder(currentOrder));
+    }
+}
+
+int InstrumentsOnScoreListModel::resolveInstrumentSequenceNumber(const QString& instrumentId) const
+{
+    for (const InstrumentTemplate* templ : repository()->instrumentTemplates()) {
```

Comment:
Better to get this value

repository()->instrumentTemplates()

only once and reuse it twice

Reply:
I see why, thanks!